<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountManagementWindow</class>
 <widget class="QDialog" name="AccountManagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QPushButton" name="changePasswordPushButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="changePrivacyPushButton">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="newPasswordLineEdit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>poil2cul</string>
     </property>
     <property name="maxLength">
      <number>256</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Password :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="currentPasswordLineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>poil</string>
     </property>
     <property name="maxLength">
      <number>256</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="loginLabel">
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current Password :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="loginTitleLabel">
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Account options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Privacy</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="publicRadioButton">
        <property name="text">
         <string>Public</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="privateRadioButton">
        <property name="text">
         <string>Private</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="serverResponseLabel">
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>currentPasswordLineEdit</tabstop>
  <tabstop>newPasswordLineEdit</tabstop>
  <tabstop>changePasswordPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
